### DISCUSSION TITLE
Why do GitHub education need my GitHub user profile to match with my school I'd?

### DISCUSSION THREAD

**Discussion by NaviTheCoderboi** [community/community#56181] [GitHub Education]
### Hi everyone,

When i applied for GitHub education, i got disapproved and this reason
```
You are unlikely to be verified until you have completed your GitHub user profile with your full name as it appears in your academic affiliation document plus a short bio
```
Why do my GitHub profile need to match with my school I'd?

**Comment by NaviTheCoderboi** [comment]
I even tried again and got this , why is this happening even i provided it my school I'd card and other real details 
![IMG_20230523_090150](https://github.com/community/community/assets/102732282/005a36d0-9e01-4a03-92f1-3f68a9598df1)

**Comment by jxpsert** [comment]
Why does it need it? Because it’s for students. And if you’re not a student, you shouldn’t have it.

**Reply by NaviTheCoderboi** [reply]
I am student and i even have modified my profile now , and provided it my school ID picture but still i am getting rejected i am 14 and i am in 9th grade

**Comment by samus-aran** [comment]
HI @NaviTheCoderboi 

Thank you for feedback and raising your concerns! We will pass this along to our internal team.

We are extremely proud that the GitHub Global Campus has provided millions of students with access to some of the industry’s most amazing tools and offers all free of charge to compliment their studies. As you can imagine there are many types of academic verification and our system is constantly evolving to accommodate this without setting the bar too high. 
However, sometimes we may require a little more evidence from the student applying to help us verify their academic status. This can be due to the system detecting a pattern used by bad actors or simply processing a new type of evidence. While submitting requested information is voluntary, we do thank students for providing this if possible as it speeds up their verification.

While we do take our Community’s feedback seriously we are also mindful to ensure that bad actors impersonating students don't jeopardize our ability to provide academic benefits of value. One aspect of this protection is momentarily compare the evidence provided against your GitHub account information.

I can confirm that once you have submitted your application you can revert this information as it's only required upon each new submission. For more information on how GitHub processes and uses personal data, please see the [GitHub Privacy Statement](https://docs.github.com/en/site-policy/privacy-policies/github-privacy-statement).

**Comment by williamregal** [comment]
[Certification Exam Dumps](https://www.ace4sure.com/](https://www.ace4sure.com/CPQ-Specialist-questions.html) are pivotal milestones in one's professional journey, and the choice of preparation platform can significantly impact success. In the dynamic realm of online learning, Ace4sure emerges as a beacon for aspiring professionals, offering a unique blend of quality, accessibility, and innovation.
